src/dune/cram_exec.ml

```diff
@@ -382,3 +382,7 @@ let run ~env ~script lexbuf : string Fiber.t =
 
 let run ~env ~script =
   run_expect_test script ~f:(fun lexbuf -> run ~env ~script lexbuf)
+
+let () = Action_exec.cram_run := Some run
```

Comment:
What about making this an `Fdecl` instead?

Reply:
Good idea, I've done that.